src/components/ItemRecognition/ItemRecognition.js

```diff
@@ -47,9 +51,10 @@ class ItemRecognition extends Component {
   handleImg = img => {
     this.model.predict(img).then(async item => {
       if (
-        item.value > ML_THRESHOLD &&
-        item.id !== '' &&
-        !this.props.prediction
+        (item.value > ML_THRESHOLD &&
+          item.id !== '' &&
+          !this.props.prediction) ||
+        (Date.now() - this.scanningStartTime) / 1000 > TIMEOUT_IN_SECONDS
```

Comment:
might need to change logic of model here, because this can return unknown ( will need to pick next best guess)

Reply:
Which part will return unknown?